vertx-sql-client/src/main/java/io/vertx/sqlclient/SqlCredentialsProvider.java

```diff
@@ -0,0 +1,36 @@
+package io.vertx.sqlclient;
+
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+
+public interface SqlCredentialsProvider {
+
+  class Static implements SqlCredentialsProvider {
```

Comment:
This is an implementation detail that shall be moved to the `impl` package.

Reply:
Ok.